frb_codegen/src/parser/ty.rs

```diff
@@ -349,7 +379,7 @@ impl<'a> TypeParser<'a> {
                 },
             })
             .collect();
-        IrEnum::new(name, wrapper_name, path, comments, variants)
+        IrEnum::new(name, wrapper_name, path, comments, variants, is_exception)
```

Comment:
TODO(fzyzcjy): Review the remaining code (mainly implementation details of frb_codegen/src/)